homework/index.js

```diff
@@ -29,19 +30,79 @@
     });
     return elem;
   }
+  function app(repos) {
+    const root = document.getElementById('root');
+    const container = createAndAppend('div', root, {
+      class: 'container',
+    });
+    const header = createAndAppend('header', container, {
+      class: 'header',
+    });
+    createAndAppend('p', header, {
+      text: 'HYF Repositories',
+    });
+    const select = createAndAppend('select', header, {
+      id: 'list',
+    });
+    const div = createAndAppend('div', root, { class: 'list-div' });
+    const list = createAndAppend('ul', div, { class: 'list1' });
+    const listContributor = createAndAppend('ul', div, { class: 'list2' });
+    const newList = createAndAppend('li', list, { text: ' ' });
+    const newList1 = createAndAppend('li', list, { text: ' ' });
+    const newList2 = createAndAppend('li', list, { text: ' ' });
+    const newList3 = createAndAppend('li', list, { text: ' ' });
 
+    for (let i = 0; i < repos.length; i++) {
+      createAndAppend('option', select, { text: repos[i].name, value: i });
+    }
+
+    function leftHand() {
+      newList.innerHTML = `Repository: <a target=_blank href= ${repos[select.value].html_url}>${
```

Comment:
Modern browsers will add the quotes for you `target=_blank`, but some old browsers might not, so do not forget to add quotes! `target="_blank"` around all the different tags.

Reply:
Thanks! I will never forget it :)